cocotb/scheduler.py

```diff
@@ -233,8 +233,8 @@ def __init__(self):
         # Our main state
         self._mode = Scheduler._MODE_NORMAL
 
-        # A list of pending (write_func, args)
-        self._write_calls = []
+        # A dictionary of pending (write_func, args)
```

Comment:
Let's expand this comment so someone doesn't try to make the same simplification as me.

```suggestion
        # A dictionary of pending (write_func, args), one per handle. Only the final write in a timestep
        # propagates to the simulator, all the rest are discarded in python.
```

Reply:
(or any similar wording, mine isn't particularly good).